GameData/RealismOverhaul/RO_RecommendedMods/Procedurals/RO_ProceduralParts.cfg

```diff
@@ -628,7 +628,7 @@
 
 	@title = Procedural Tank (Service Module)
 	@manufacturer = Generic
-	@description = Specialized procedural tank for service modules. There are different levels of service modules that you can unlock. Each tank type has a different base mass, different cost, and different amounts of utilization it can have. These are the only types of tanks that can have life support resources added to them.
+	@description = Specialized procedural tank for batteries and life support, not intended to hold fuel. There are different levels of service modules that you can unlock. Each tank type has a different base mass, different cost, and different amounts of utilization it can have. These are the only types of tanks that can have life support resources added to them.
```

Comment:
If you're going for less reading, then removing the redundant 'procedural' might be warranted.